src/filters/Filter.ts

```diff
@@ -69,10 +69,11 @@ export interface FilterOptions
     blendMode?: BLEND_MODES;
     /**
      * the resolution the filter should be rendered at. The lower the resolution, the more performant
-     * the filter will be, but the lower the quality of the output. (defaults to the renderers resolution)
+     * the filter will be, but the lower the quality of the output. (default 1)
+     * If `null` the resolution of the render target is used.
      * Consider lowering this for things like blurs filters
      */
-    resolution?: number;
+    resolution?: number | null;
```

Comment:
Or do we prefer `number | 'inherit'`?

Reply:
I like inherit, it's easier to reason than null.

Reply:
Made the change.